lib/fog/utilities.rb

```diff
@@ -1,7 +1,9 @@
 class Hash
   def symbolize_keys!
     keys.each do |key|
-      self[(key.to_sym rescue key)] = delete(key) if key.respond_to?(:to_sym) && !key.is_a?(Fixnum)
+      if key.respond_to?(:to_sym) && !key.is_a?(0.class)
+        self[(key.to_sym rescue key)] = delete(key)
```

Comment:
This could surely just be replaced with `key.to_sym` since you have already checked it with `respond_to?`.